server/schedule/checker_controller.go

```diff
@@ -70,11 +72,26 @@ func (c *CheckerController) CheckRegion(region *core.RegionInfo) []*operator.Ope
 
 	if c.opts.IsPlacementRulesEnabled() {
 		if op := c.ruleChecker.Check(region); op != nil {
+			added := false
+			// ignore spilt
+			if op.Kind()&operator.OpReplica != 0 {
+				missPeers, count := c.GetRuleChecker().GetMissPeer(region)
```

Comment:
This calculate is Repeated, maybe we can record the information in the operator?

Reply:
ok, it sounds good